datadog/plugins/mongodb.sls

```diff
@@ -0,0 +1,18 @@
+{% set dd_mongo = salt.pillar.get(
+    'datadog:integrations:mongodb',
+    {
+        'passwd': 'Ch@ng3me!',
+        'password': 'N0tS3cure!'
+    }) %}
```

Comment:
It seems worth adding entries for these in the `pillar.example`, especially the one corresponding with the datadog "passwd" - since it makes it harder for someone to accidentally only add their admin "password" and wind up keeping the other default.